xata/helpers.py

```diff
@@ -315,12 +317,12 @@ def __init__(
         self.has_run = False
         self.operations = {"operations": []}
 
-    def _add_operation(self, operation: dict):
+    def _add_operation(self, operation: dict) -> None:
         if len(self.operations["operations"]) >= TRX_MAX_OPERATIONS:
             raise Exception(f"Maximum amount of {TRX_MAX_OPERATIONS} transaction operations exceeded.")
         self.operations["operations"].append(operation)
 
-    def insert(self, table: str, record: dict, createOnly: bool = False):
+    def insert(self, table: str, record: dict, createOnly: bool = False) -> None:
```

Comment:
python noob question: 

What does the `-> None` do?

Reply:
Sure :) It's equivalent to `void` in C or Java, it indicates that the method returns nothing.

Reply:
I've never seen type annotation for Python, I assume this is a relatively new thing?

Reply:
Yea, I think they introduce type hints in `3.8` (I think). It makes implementation more convenient.